### PR TITLE
Adds a blog post for Serf 0.7.

### PR DESCRIPTION
**Pull request by slackpad**

### CODE REVIEW

File: source/blog/2015-12-22-serf-0.7.html.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+page_title: "Serf 0.7"
+title: "Serf 0.7"
+list_image_url: "/images/blog/serf/serf-logo-red.png"
+tags: serf
+author: James Phillips
+---
+
+Today we are releasing [Serf 0.7](http://www.serfdom.io). Serf is a decentralized
+solution for cluster membership, failure detection, and orchestration.
```
Comment (mitchellh) [inline source/blog/2015-12-22-serf-0.7.html.md]: Sentence after this? 

Serf is in use in some huge deployments (more than 10,000 machines in a single cluster!), and powers Consul and Nomad.

File: source/blog/2015-12-22-serf-0.7.html.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+Serf 0.7 has been designed to "upshift" automatically as nodes are upgraded to
+start taking advantage of network tomography features and new protocol features
+such as the TCP fallback probe. For most configurations, upgrading will just
+require an agent restart with the new binary.
+
+More details are available on the [upgrade process here](https://serfdom.io/docs/upgrading.html).
+
+## Roadmap
+
+The next release will be focused on more sophisticated algorithms for node
```
Comment (mitchellh) [inline source/blog/2015-12-22-serf-0.7.html.md]: Prefix paragraph:

Serf is extremely stable. It receives very few bug reports, despite powering Consul and Nomad in addition to being widely deployed itself. 
Despite its complexity, Serf is one of our most stable pieces of software. It powers Consul and Nomad, ra

**Comment by mitchellh** [comment]
Added a little bit more flare. Actually content looks great.

**Comment by slackpad** [comment]
@mitchellh thanks for the review!

**Pull request merged**